opts/opts_test.go

```diff
@@ -28,6 +28,20 @@ func TestValidateIPAddress(t *testing.T) {
 
 }
 
+func TestValidateMACAddress(t *testing.T) {
+	if _, err := ValidateMACAddress(`92:d0:c6:0a:29:33`); err != nil {
+		t.Fatalf("ValidateMACAddress(`92:d0:c6:0a:29:33`) got %s", err)
+	}
+
+	if _, err := ValidateMACAddress(`92:d0:c6:0a:33`); err == nil {
+		t.Fatalf("ValidateMACAddress(`92:d0:c6:0a:33`) got %s", err)
```

Comment:
`got %s` unnecessary as `err == nil` in this case; I would change message to "ValidateMACAddress(..) succeeded; expected failure on invalid MAC"